uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/category/CategorySelectorView.kt

```diff
@@ -107,7 +107,7 @@ class CategorySelectorView @JvmOverloads constructor(context: Context, attrs: At
     }
 
     override fun showCategories() {
-        this.visibility = View.VISIBLE
+        this.visibility = View.GONE
```

Comment:
At the end we should circle back and delete anything quote list related that is no longer being used